pages/k8s/azure-cloud-provider.md

```diff
@@ -0,0 +1,203 @@
+---
+wrapper_template: "templates/docs/markdown.html"
+markdown_includes:
+  nav: "kubernetes/docs/shared/_side-navigation.md"
+context:
+  title: "Charmed Kubernetes on Azure"
+  description: Running Charmed Kubernetes on Azure using the azure-cloud-provider.
+keywords: azure, integrator, cloud-provider
+tags: [install]
+sidebar: k8smain-sidebar
+permalink: azure-integration.html
+layout: [base, ubuntu-com]
+toc: False
+---
+
+**Charmed Kubernetes** will run seamlessly on Microsoft Azure<sup>&reg;</sup>.
+With the addition of the `azure-cloud-provider`, your cluster will also be able
+Azure native features as defined by the out-of-tree definitions.
+
+
+## Azure Cloud Provider
+
+The [`azure-cloud-provider`][azure-cloud-provider] charm gathers 
+credentials from the [`azure-integrator`][azure-integrator] charm, and deploys
+configurable versions of the [cloud-provider][] and [azure-disk][] into the
+cluster rather than using the built-in cloud-providers within the kubernetes binaries.
+After installation, the cluster can identify each node's providerID, create loadbalancers, and create AzureDisk volumes.
+
+### Installing
+
+If you install **Charmed Kubernetes** [using the Juju bundle][install],
+you can add the azure-cloud-provider at the same time by using the following
+overlay file ([download it here][asset-azure-cloud-overlay]):
+
+```yaml
+description: Charmed Kubernetes overlay to add native Azure support.
+applications:
+  kubernetes-control-plane:
+    options:
+      register-with-taints: ""
```

Comment:
```suggestion
```

small tweak to bring in-line with [the overlay](https://github.com/charmed-kubernetes/bundle/pull/854/files)